src/ViewHelpers/DisplayApplicantViewHelper.php

```diff
@@ -19,12 +19,30 @@ public static function displayApplicants($applicants)
         foreach ($applicants as $applicant) {
             if ($applicant instanceof BaseApplicantEntityInterface) {
                 $result .= '<tr>
-                        <td><a data-id ="'. $applicant->getId().'" type="button"  class="myBtn">'
-                            . $applicant->getName() .'</a></td>
-                            <td>'. $applicant->getEmail() .'</td>
-                            <td>'. $applicant->getDateOfApplication() .'</td>
-                            <td>'. $applicant->getCohortDate().'</td>
-                        </tr>';
+                                <td>
+                                    <a data-id ="' . $applicant->getId() . '" 
+                                       type="button"  
+                                       class="myBtn">
+                                      ' . $applicant->getName() . '
+                                    </a>
+                                </td>
+                                <td>' . $applicant->getEmail() . '</td>
+                                <td>' . $applicant->getDateOfApplication() . '</td>
+                                <td>' . $applicant->getCohortDate() . '</td>
+                                <td>                              
+                                    <a href="/editApplicant?id=' . $applicant->getId() . '"   
```

Comment:
the butotns dont sit next to each other. can we make the table full width do that they do?